src/dominion/supply.rs

```diff
@@ -40,3 +40,37 @@ impl Supply {
         }
     }
 }
+
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    fn opt_to_bool<T>(opt: &Option<T>) -> bool {
```

Comment:
You can use is_some or is_none:
https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some

Reply:
Thanks, done.